pypot/server/httpserver.py

```diff
@@ -203,8 +203,10 @@ def call_primitive_method(prim, meth):
                 '{}:{}'.format(prim, meth): res
             }
 
-    def run(self, quiet=False, server='tornado'):
+    def run(self, quiet=None, server='tornado'):
         """ Start the bottle server, run forever. """
+        if quiet is None:
+            quiet = self.quiet
```

Comment:
quiet = self.quiet if quiet is None else quiet

You know... just because you can ^^